### DISCUSSION TITLE
Transfer LCAO wfc to pw basis

### DISCUSSION THREAD

**Discussion by hongriTianqi** [deepmodeling/abacus-develop#5291] [General]

**Comment by hongriTianqi** [comment]
This topic is mentioned by @mohanchen.

**Comment by Fadelis98** [comment]
The inverse transform that transforms pw wfc to lcao basis would be very useful as well, but it now suffers for low accuracy, did you plan to implement it?  I investigated this topic before, and I would recommand this paper: [https://pubs.aip.org/aip/jcp/article/143/10/104109/898790/A-projection-free-method-for-representing-plane](url)

**Comment by hongriTianqi** [comment]
@mohanchen Do you have comments on this issue? thanks.

**Comment by mohanchen** [comment]
I will think about it
